.devcontainer/Dockerfile

```
@@ -22,40 +15,13 @@ ENV PYTHONPATH="${PWD}/build/lib.linux-x86_64-3.9:${PYTHONPATH}"
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
-    ca-certificates \
     clang-format \
```

gdb、cppにはあるがpythonに無いみたいなので、追加してもいいかなと思いました

```suggestion
    clang-format \
    gdb \
```